department-of-veterans-affairs/caseflow
# Queue | Always scroll to top of page after navigating

Resolves #5293 

### Description
Because Queue is a single-page app, the browser does not automatically reset scroll position when navigating between 'pages.' This scrolls to the top of the page every time the route changes (within `QueueApp`'s `BrowserRouter`)

### Acceptance Criteria 
- [ ] Browser scrolls to top of page on route change

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE`. Scroll to bottom of page, click to go to case details with many tasks (more than one page-length)
2. Confirm page scrolls back to top of page on route change
